DQM/Integration/python/clients/pixel_dqm_sourceclient-live_For_Timing_Scan_cfg.py

```diff
@@ -0,0 +1,182 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
@dmitrijus I have removed the file in the DQM/Integration directory and renamed it properly in our SiPixelPhase1/Config folder. In the case we will decide to use it Online I will make a separate PR. It is important to have this merged as soon as possible, in the way new contributors can play with a "quasi-final" version of the code without struggling with private branche etc ...